src/e2e/specs/auth.spec.ts

```diff
@@ -27,10 +27,7 @@ test.describe(`${process.env.E2E_TEST_ENV} - Authentication flow verification @s
     await authPage.signUp(randomEmail, page);
 
     // assert successful login
-    const successUrl =
-      process.env.E2E_TEST_ENV === "local"
-        ? "/user/dashboard"
-        : "/user/welcome";
+    const successUrl = "/user/dashboard";
```

Comment:
Is this still necessary? Eventually, the tests will need to be updated to account for the welcome page as well, I imagine.

Actually, I guess we'll want tests for both the "pre-redesign user" and "new user" experience.

Reply:
I think this needs to be updated either way but should be good to leave this as is for now. Also updated in [eb173c9](https://github.com/mozilla/blurts-server/pull/4174/commits/eb173c9828365e8f15dc7a42f030655a77317545).